roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -138,6 +138,10 @@ cpu_mode = host-passthrough
 {% else %}
 cpu_mode = host-model
 {% endif -%}
+live_migration_downtime = 500
+live_migration_downtime_steps = 10
+live_migration_downtime_delay = 75
+live_migration_uri = "qemu+ssh://%s/system?keyfile=/opt/stack/data/nova/.ssh/id_rsa"
```

Comment:
This may need to point at the `known_hosts` file that's also in /opt/stack/data/nova/.ssh/

Also, this path should use the variable, `{{ nova.state_path }}/.ssh/`